docs/Developers/Newcomers_Guide.rst

```diff
@@ -333,6 +333,11 @@ and clicking ``New Pull Request``.
     updated, you can edit your comment instead. If you need to add new
     information, make a new comment.
 
+    If you are generating a pull request for a particular issue, you need to
+    specify the issue number that your pull request resolves. If an issue is
+    labeled as a ``bug``, use ``Fixes XXXX`` and if it's not labeled as
+    ``bug``, ``use Closes XXXX``, where ``XXXX`` denotes the issue number.
```

Comment:
1. "denotes the issue number": Try to say "denotes the full issue URL" :+1:
2. Maybe change the format to `Fixes <full-issue-url>`/`Closes <full-issue-url>` or similar?

Reply:
@Makman2 Okay, thanks for the review. I will try to correct the things.

Reply:
Sorry it took so long, sometimes PRs are missed :/ In that case feel free to ping after a while so we get new notifications ;)

Reply:
@Makman2 I have done the required changes, please see and guide if any more corrections are required. Also, as you proposed to update Writing_Good_Commits.rst, so, should I open a new PR for that now?